src/bbs2gh/Commands/GenerateScriptCommand.cs

```diff
@@ -50,7 +52,8 @@ public class GenerateScriptCommand : CommandBase<GenerateScriptCommandArgs, Gene
 
     public Option<string> BbsSharedHome { get; } = new(
         name: "--bbs-shared-home",
-        description: "Bitbucket server's shared home directory. If not provided \"/var/atlassian/application-data/bitbucket/shared\" will be used.");
+        description: "Bitbucket server's shared home directory. If not provided \"/var/atlassian/application-data/bitbucket/shared\" will be used when using SSH to download the export archive " +
```

Comment:
What do you think of rephrasing this more like this? I think it might be easier to understand.

> Bitbucket Server's shared home directory, where migration archives are stored. Defaults to `/var/atlassian/application-data/bitbucket/shared\` if downloading the archive from the server using SSH, and `c$\\atlassian\\applicationdata\\bitbucket\\shared\` if downloading using SMB.

Reply:
For my suggestions in this file, if you accept them, we should apply them to `src/bbs2gh/Commands/MigrateRepoCommand.cs` too.

Reply:
I am going to modify the copy text but gonna skip `where migration archives are stored` b/c there are a lot more things stored in the shared home, basically the entire Bitbucket data is stored there including the migration archive. This should be a known term to a Bitbucket admin so I don't think we really need to add more explanation :)

Reply:
addressed in 40c97a3e9cbeed8a169e3cd1252911c167dd261b